### PR TITLE
tunnel: update to version 0.3.2

### PR DESCRIPTION
**Pull request by vishr**
Signed-off-by: Vishal Rana <vr@labstack.com>

### CODE REVIEW

File: bucket/tunnel.json [hunk 1]
```diff
@@ -1,39 +1,33 @@
...
         }
     },
-    "bin": [
-        [
-            "tunnel-client.exe",
-            "tunnel"
-        ],
-        "tunnel-client.exe"
-    ],
+    "bin": [["tunnel.exe", "tunnel"], "tunnel.exe"],
```
Comment (r15ch13) [inline bucket/tunnel.json]: Could reduced to:
```suggestion
    "bin": "tunnel.exe",
```

File: bucket/tunnel.json [hunk 2]
```diff
@@ -1,39 +1,33 @@
...
+                "url": "https://github.com/labstack/tunnel-client/releases/download/v$version/tunnel_$version_windows_32-bit.zip"
             },
             "64bit": {
-                "url": "https://github.com/labstack/tunnel-client/releases/download/v$version/tunnel-client_$version_windows_64-bit.zip"
+                "url": "https://github.com/labstack/tunnel-client/releases/download/v$version/tunnel_$version_windows_64-bit.zip"
             }
         },
         "hash": {
-            "url": "https://github.com/labstack/tunnel-client/releases/download/v$version/tunnel-client_checksum.txt"
+            "url": "https://github.com/labstack/tunnel-client/releases/download/v$version/checksum.txt"
```
Comment (Ash258) [inline bucket/tunnel.json]:
```suggestion
            "url": "$baseurl/checksum.txt"
```

Comment (vishr) [reply bucket/tunnel.json]: @Ash258 Can this be applied to architecture block as well?

Comment (Ash258) [reply bucket/tunnel.json]: What?

Comment (vishr) [reply bucket/tunnel.json]: NV

File: bucket/tunnel.json [hunk 3]
```diff
@@ -1,39 +1,33 @@
...
     },
-    "bin": [
-        [
-            "tunnel-client.exe",
-            "tunnel"
-        ],
-        "tunnel-client.exe"
-    ],
+    "bin": "tunnel.exe",
     "checkver": {
```
Comment (Ash258) [inline bucket/tunnel.json]:
```suggestion
    "checkver": "github",
```

Comment (vishr) [reply bucket/tunnel.json]: @Ash258 Looks like something wrong in this change, can you review again?

**Pull request merged**